guide/english/accessibility/index.md

```diff
@@ -4,11 +4,11 @@ title: Accessibility
 ## Accessibility
 <strong>Web accessibility means that people with disabilities can use the Web</strong>.
 
-More specifically, Web accessibility means that people with a wide range of disabilities can perceive, understand, navigate, and interact with the Web, and that they can contribute to the Web. Web accessibility can make it easier for people to find a website, access it, and use it successfully, thus resulting in increased audience (more users) and increased effectiveness (more use).
+More specifically, Web accessibility means that people with a wide range of disabilities can perceive, understand, navigate, and interact with the Web and that they can contribute to the Web. Web accessibility can make it easier for people to find a website, access it, and use it successfully, thus resulting in an increased audience (more users) and increased effectiveness (more useful).
 
 Web accessibility encompasses all disabilities that affect access to the Web, including visual, auditory, physical, speech, cognitive, and neurological disabilities. The article [How People with Disabilities Use the Web](http://www.w3.org/WAI/intro/people-use-web/Overview.html) describes how different disabilities affect Web use and includes scenarios of people with disabilities using the Web.
 
-Web accessibility also **benefits** people *without* disabilities. For example, a key  principle of Web accessibility is designing Web sites and software that are flexible to meet different user needs, preferences, and situations. This **flexibility** also benefits people *without* disabilities in certain situations, such as people using a slow Internet connection, people with "temporary disabilities" such as a broken arm, and older people with changing abilities due to aging. The article [Developing a Web Accessibility Business Case for Your Organization](https://www.w3.org/WAI/bcase/Overview) describes many different benefits of Web accessibility, including **benefits for organizations**.
+Web accessibility also **benefits** people *without* disabilities. For example, a key principle of Web accessibility is designing Web sites and software that are flexible to meet different user needs, preferences, and situations. This **flexibility** also benefits people *without* disabilities in certain situations, such as people using a slow Internet connection, people with "temporary disabilities" such as a broken arm, and older people with changing abilities due to ageing. The article [Developing a Web Accessibility Business Case for Your Organization](https://www.w3.org/WAI/bcase/Overview) describes many different benefits of Web accessibility, including **benefits for organizations**.
```

Comment:
We use the American version of English for this project so `aging` is correct, please revert the change